frontend/packages/dev-console/src/components/helm/__tests__/HelmReleaseDetailsPage.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { ErrorPage404 } from '@console/internal/components/error';
+import { DetailsPage } from '@console/internal/components/factory';
+import HelmReleaseDetailsPage from '../HelmReleaseDetailsPage';
+
+type HelmReleaseDetailsPageProps = React.ComponentProps<typeof HelmReleaseDetailsPage>;
+let helmReleaseDetailsPageProps: HelmReleaseDetailsPageProps;
```

Comment:
No need to define a props variable as it use only once. 
```suggestion
let helmReleaseDetailsPageProps: React.ComponentProps<typeof HelmReleaseDetailsPage>;
```